packages/itwin/grouping-mapping-widget/src/components/Mappings/MappingsView.tsx

```diff
@@ -60,6 +61,7 @@ export interface MappingsViewProps {
   onClickAddMapping?: () => void;
   onClickMappingTitle?: (mapping: Mapping) => void;
   onClickMappingModify?: (mapping: Mapping) => void;
+  onRunExtraction: (mappings: Mapping[]) => Promise<void>;
```

Comment:
Adding a mandatory prop to a component that is exported from the package is a breaking change.

Reply:
I have moved this inside `MappingView`.